controlpanel/frontend/jinja2/release-create.html

```diff
@@ -0,0 +1,109 @@
+{% set legend -%}
+  Create a new release of a tool in the analytical platform.
+{%- endset %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">{{ page_title }}</h1>
+
+  {% if form.error %}
+    <div class="error">{{ form.error }}</div>
+  {% endif %}
+
+  <form action="{{ url('create-release') }}" method="post">
+    {{ csrf_input }}
+    {% call govukFieldset({ "legend": { "html": legend|safe } }) %}
+    {{ govukInput({
+      "label": {
+        "text": "Name",
+        "classes": "govuk-label--m",
+      },
+      "classes": "govuk-!-width-two-thirds",
+      "hint": {
+        "text": 'Human readable name. 60 chars max'
+      },
+      "name": "name",
+      "attributes": {
+        "maxlength": "60",
+      },
+      "value": form.name.value(),
+      "errorMessage": { "html": form.name.errors|join(". ") } if form.name.errors else {}
+    }) }}
+    {{ govukInput({
+      "label": {
+        "text": "Chart Name",
+        "classes": "govuk-label--m",
+      },
+      "classes": "govuk-!-width-two-thirds",
+      "hint": {
+        "text": 'Helm chart name. 60 chars max, only lowercase letters and hyphens'
+      },
+      "name": "chart_name",
+      "attributes": {
+        "pattern": "[a-z0-9.-]{1,60}",
+        "maxlength": "60",
+      },
+      "value": form.chart_name.value(),
+      "errorMessage": { "html": form.chart_name.errors|join(". ") } if form.chart_name.errors else {}
+    }) }}
+    {{ govukInput({
+      "label": {
+        "text": "Version",
+        "classes": "govuk-label--m",
+      },
+      "classes": "govuk-!-width-two-thirds",
+      "hint": {
+        "text": 'Helm chart version. Only lowercase letters, numbers, periods and hyphens'
+      },
+      "name": "version",
+      "attributes": {
+        "pattern": "[a-z0-9.-]{1,60}",
+        "maxlength": "60",
+      },
+      "value": form.version.value(),
+      "errorMessage": { "html": form.version.errors|join(". ") } if form.version.errors else {}
+    }) }}
+    {{ govukCheckboxes({
+      "fieldset": {
+        "legend": {
+          "text": "Restricted Release",
+          "classes": "govuk-fieldset__legend--m",
+        },
+      },
+      "classes": "govuk-!-width-two-thirds",
+      "hint": {
+        "text": 'A flag to indicate if the release is a release restricted to certain users'
+      },
+      "name": "is_restricted",
+      "items": [
+        {
+          "value": "True",
+          "text": "Is a Restricted Release",
+          "checked": form.is_restricted.value(),
+        },
+      ],
+      "errorMessage": { "html": form.is_restricted.errors|join(". ") } if form.is_restricted.errors else {}
+    }) }}
+    {{ govukInput({
+      "label": {
+        "text": "Target Users",
+        "classes": "govuk-label--m",
+      },
+      "classes": "govuk-!-width-two-thirds",
+      "hint": {
+        "text": 'A comma separated list of usernames of usernames of users who have access to this release'
```

Comment:
duplicated "of usernames of usernames", glad I'm not the only one glitching and writing things like this 😝

Reply:
What do you what do you mean mean about glitching out out glitching..? :stuck_out_tongue_closed_eyes: Nothing wrong wrong with that.

(Seriously, good catch, I'll fix.)

Reply:
Fixed fixed